.changeset/strong-mayflies-shake.md

```diff
@@ -0,0 +1,6 @@
+---
+'@westpac/popover': major
+---
+
+Added blender support and the `placement` prop
+You can now use `placement` to override the position detection for the popover panel.
```

Comment:
please remove trailing `.` ;)

Reply:
Not a sentence?

Reply:
Individual bullets don't get full stops, unless the bulleted item has 2+ sentences

- I am a thing
- I am two things. See how I have two full stops.

Reply:
I don't see bullet points though? I guess the first sentence is missing a period?

Reply:
Sorry thought the individual lines got turned into bullets in the resulting changelog file

Reply:
Oh do they? I don't know! :D

Reply:
Don't think so. Mind adding a training `.` after the word 'prop' (line5)